multi-int-store/src/main/java/com/connexta/multiintstore/common/StorageManager.java

```diff
@@ -50,9 +50,6 @@ private void sortCallbacks(Object callback, String ingestId) {
   }
 
   public void handleGeneralCallback(Object callback, String ingestId) {
```

Comment:
✏️ Make it clear that Exceptions from this method should be caught. Same for `#handleCallback(MetadataCallback, String)`